src/js/components/container/ButtonContainer.js

```diff
@@ -29,8 +60,15 @@ class ButtonContainer extends Component {
           classes="btn btn-warning"
           text="Change Color"
           id="ColorButton"
-          onClickHandler={this.onClickHandler}
+          onClickHandler={this.handleClick}
         />
+        { this.state.displayColorPicker ? <div style={ popover } >
+          <div style={ cover } onClick={ this.handleClose }/> 
+          <CompactPicker
+          color = { this.state.color }
```

Comment:
Can you fix the indenting here please =D

Reply:
On it!